login/template.ftl

```diff
@@ -126,13 +127,21 @@
 			margin-top: 24px;
 			color: #777;
 		}
+        #kc-site-name {
+            font-size: 2em;
+            color: #FFF;
+            text-align: center;
+            text-transform: capitalize;
+            margin-top: 8px;
+        }
 	</style>
 </head>
 
 <body class="${properties.kcBodyClass!}">
   <div class="${properties.kcLoginClass!}">
     <div id="kc-header" class="${properties.kcHeaderClass!}">
       <div id="kc-header-wrapper" class="${properties.kcHeaderWrapperClass!}">${kcSanitize(msg("loginTitleHtml",(realm.displayNameHtml!'')))?no_esc}</div>
+      <div id="kc-site-name">${__KEYCLOAK_SITE_NAME}</div>
```

Comment:
I think it needs to be a [special format](https://www.keycloak.org/docs/latest/server_development/index.html#theme-properties) for the FTL templates?

Will try out on shared testing first either way
```suggestion
      <div id="kc-site-name">${env.__KEYCLOAK_SITE_NAME}</div>
```

Reply:
thank you, Ivan.  Fixed